dev/dll/SharedHelpers.cpp

```diff
@@ -807,3 +807,23 @@ winrt::float4 SharedHelpers::RgbaColor(const winrt::Color& color)
 {
     return { static_cast<float>(color.R), static_cast<float>(color.G), static_cast<float>(color.B), static_cast<float>(color.A) };
 }
+
+bool SharedHelpers::IsFocusable(winrt::DependencyObject const& object, bool checkTabStop)
+{
```

Comment:
There are elements other than Control that can be focused such as Hyperlink and WebView.

I think it is ok to not handle them here, but maybe add a comment saying so.

Reply:
Sounds good.